### PR TITLE
TechPowerUp.GPU-Z version 2.41.0

### PR DESCRIPTION
**Pull request by KaranKad**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24084)

### CODE REVIEW

**Review by OfficialEsco — COMMENTED** [review]
```
Found TechPowerUp GPU-Z [TechPowerUp.GPU-Z]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://us2-dl.techpowerup.com/files/P5KzyM4MeQmuwNFu5Zz5RA/1628268084/GPU-Z.2.41.0.exe
An unexpected error occurred while executing the command:
Download request status is not success.
0x8019019a : Gone (410).
```

image
>:c

File: manifests/t/TechPowerUp/GPU-Z/2.41.0/TechPowerUp.GPU-Z.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://us2-dl.techpowerup.com/files/P5KzyM4MeQmuwNFu5Zz5RA/1628268084/GPU-Z.2.41.0.exe
```
Comment (OfficialEsco) [inline manifests/t/TechPowerUp/GPU-Z/2.41.0/TechPowerUp.GPU-Z.installer.yaml:16]:
```suggestion
  InstallerUrl: https://us2-dl.techpowerup.com/files/Oh0FkuN6vImBhwXIQwpZLQ/1628272541/GPU-Z.2.41.0.exe
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Does Techpowerup not have a Auto link?

**Review by OfficialEsco — APPROVED** [review]

**Comment by W1zzardTPU** [comment]
GPU-Z author here, what HTTP user agent does winget use? I can whitelist it on our download servers, so the direct download link doesn't expire

**Comment by OfficialEsco** [comment]
@denelon @palenshus

**Comment by denelon** [comment]
@W1zzardTPU I'll get someone to post it here for you. We should also update our docs to make it easier for others to find out as well.

**Comment by ashpatil-msft** [comment]
We use "winget/1.0" as the User agent.

**Comment by OfficialEsco** [comment]
@KaranKad was smarter than me and already used https://us2-dl.techpowerup.com/files/GPU-Z.2.41.0.exe :)

**Comment by maver1ck** [comment]
@W1zzardTPU 
Is there a chance for similar workaround for Scoop ?

**Comment by W1zzardTPU** [comment]
> 
> 
> @W1zzardTPU
> Is there a chance for similar workaround for Scoop ?

Thanks, done

**Pull request merged**